searchcore/src/tests/proton/feedoperation/feedoperation_test.cpp

```diff
@@ -311,15 +311,15 @@ TEST_F("require that we can serialize and deserialize remove operations", Fixtur
     uint32_t expSerializedDocSize = getDocIdSize(docId);
     EXPECT_NOT_EQUAL(0u, expSerializedDocSize);
     {
-        RemoveOperation op(bucket, Timestamp(10), docId);
+        RemoveOperationWithDocId op(bucket, Timestamp(10), docId);
         op.setDbDocumentId({1, 2});
         op.setPrevDbDocumentId({3, 4});
         EXPECT_EQUAL(0u, op.getSerializedDocSize());
         op.serialize(stream);
         EXPECT_EQUAL(expSerializedDocSize, op.getSerializedDocSize());
     }
     {
-        RemoveOperation op;
+        RemoveOperationWithDocId op;
```

Comment:
Please extend unit test to also test (de)serialize of RemoveOperationWithGid.

Reply:
Test added.